webrtc.html

```diff
@@ -5271,8 +5271,8 @@ <h2 id="sec.cert-mgmt">Certificate Management</h2>
     readonly        attribute RTCDtlsTransport?  transport;
     readonly        attribute RTCDtlsTransport? rtcpTransport;  // Feature at risk
     static RTCRtpCapabilities getCapabilities (DOMString kind);
-    Promise&lt;void&gt;             setParameters (optional RTCRtpParameters parameters);
-    RTCRtpParameters          getParameters ();
+    Promise&lt;void&gt;             setParameters (RTCRtpSenderParameters parameters);
```

Comment:
This needs to be optional. It's a weird WebIDL thing. https://github.com/w3c/webrtc-pc/pull/419